conf/messages.en

```diff
@@ -242,3 +242,18 @@ isThisYourBusiness.error.required = Select yes if this is your business
 
 detailsMatched.title = We have matched your details
 detailsMatched.heading = We have matched your details
+
+soleTraderDetailsNotMatch.title = The details you entered did not match our records
+soleTraderDetailsNotMatch.heading = The details you entered did not match our records
+soleTraderDetailsNotMatch.p1 = To help us find your business, check if you have entered the right details.
+soleTraderDetailsNotMatch.p2 = Try entering your UTR again
+soleTraderDetailsNotMatch.p3 = If entering your UTR again does not work, you can
+soleTraderDetailsNotMatch.p3.link.text = find a lost UTR.
+soleTraderDetailsNotMatch.p3.link.href = https://www.gov.uk/find-utr-number
```

Comment:
This is a constant that does not need translation. You can hardcode it in the template